CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-# Contributing to OpenWhisk
+# Contributing to OpenWhisk catalog
 
 We welcome contributions, but request you follow these guidelines.
 
  - [Raising issues](#raising-issues)
+ - [Contributing your own package](#contributing-your-own-package)
  - [Contributor License Agreement](#contributor-license-agreement)
- - [Coding Standards](#coding-standards)
- 
+
 ## Raising issues
 
 Please raise any bug reports on the issue tracker. Be sure to
```

Comment:
Please raise any bug reports on the issue tracker => Please report bugs on the issue tracker